2008-08-24  Peter McCurdy <pmccurdy@skeptopotamus>

	* A stray comma between two string literals caused incorrect 
          output and a compiler warning.

diff -rpuN --exclude .depend --exclude '*~' --exclude '*.orig' --exclude '*.rej' --exclude '*.l[oa]' --exclude '*.lai' --exclude '*.o' --exclude .deps --exclude .libs --exclude Makefile --exclude 'stamp-*' --exclude messagebus --exclude rc.messagebus --exclude session.conf --exclude system.conf --exclude config.h --exclude config.log --exclude config.status --exclude dbus-1.pc --exclude Doxyfile --exclude libtool --exclude data --exclude dbus-arch-deps.h ../.build_orig/dbus/dbus/dbus-marshal-recursive.c dbus/dbus/dbus-marshal-recursive.c
--- ../.build_orig/dbus/dbus/dbus-marshal-recursive.c	2007-07-24 11:39:08.000000000 -0400
+++ dbus/dbus/dbus-marshal-recursive.c	2008-08-24 19:45:42.000000000 -0400
@@ -1652,7 +1652,7 @@ writer_recurse_init_and_check (DBusTypeW
                                      _dbus_type_to_string (expected),
                                      _dbus_string_get_const_data (writer->type_str), writer->type_pos);
           else
-            _dbus_warn_check_failed ("Writing an element of type %s, but no value is expected here\n",
+            _dbus_warn_check_failed ("Writing an element of type %s, but no value is expected here\n"
                                      "The overall signature expected here was '%s' and we are on byte %d of that signature.\n",
                                      _dbus_type_to_string (sub->container_type),
                                      _dbus_string_get_const_data (writer->type_str), writer->type_pos);
